### PR TITLE
Use Vue for share form

### PR DESCRIPTION
**Pull request by paulcsmith**

### CODE REVIEW

File: src/html/share-lease.hbs [hunk 1]
```diff
@@ -1,59 +1,59 @@
...
+          <input id="name" class="b--black-20 ba db f6 input-reset mb2 pa3 w-100" type="email" required placeholder="enter an email"
+            aria-describedby="name-desc">
         </div>
-      </div>
-    </form>
 
+        <div class="flex items-baseline justify-end mt3">
+          <a href="/view-lease" class="f6 link dim black db">Cancel</a>
+          <div class="flex flex-column ml6 pt3">
+            <button class="b--black ba bg-navy f6 grow input-reset pa3 br2 pointer white" :class="{'o-50': isSending}">
```
Comment (paulcsmith) [inline src/html/share-lease.hbs:34]: This is the import part. Sets class if it is sending

File: src/html/share-lease.hbs [hunk 2]
```diff
@@ -1,59 +1,59 @@
...
+              <span v-if="isDone">
+                <i class="material-icons material-icon-small">check</i>
+                Done
+              </span>
+              <span v-else="!isDone">\{{ buttonText }}</span>
```
Comment (paulcsmith) [inline src/html/share-lease.hbs:39]: This tells Vue what text to set depending on if it is done or not

File: src/html/share-lease.hbs [hunk 3]
```diff
@@ -1,59 +1,59 @@
...
+<share-form inline-template>
+  <div>
+    <a href="/view-lease" class="dark-gray flex items-center link pl3 pt3">
+      <i class="bg-light-gray br-100 material-icons pa3">close</i>
+    </a>
+    <h1 class="center f2 lh-title mw7 normal pb5 pt2 tc">
+      Start your lease negotiation for New Pacific Place with LeasePilot
+    </h1>
+    <div class="b--black-05 bg-near-white bt center pv4 vh-100">
+      <form action="/view-lease" class="black-80 center mw6 pa4" @submit.prevent='submit'>
```
Comment (paulcsmith) [inline src/html/share-lease.hbs:11]: Here's the submit handler

File: src/js/commentable.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    },
+    removeComment: function(index) {
+      new CommentStore(this.commentableId).remove(index);
+      this.$forceUpdate();
+    },
+    hasComments: function() {
+      return this.comments().length;
+    }
+  }
+});
```
Comment (paulcsmith) [inline src/js/commentable.js:35]: This is some cleanup that doesn't have to do with the share form

**Review by enatario — APPROVED** [review]

**Pull request merged**